File: test/fixtures/php-guzzle/put_with_file.php
<?php
require 'vendor/autoload.php';

use GuzzleHttp\Client;

$client = new Client();

$response = $client->put('http://localhost:28139/upload', [
    'headers' => [
        'Content-Type' => 'application/x-www-form-urlencoded'
    ],
    'body' => '@new_file'
]);

File: test/fixtures/java-unirest/put_with_file.java
import kong.unirest.HttpResponse;
import kong.unirest.Unirest;

class Main {
    public static void main(String[] args) {
            HttpResponse<String> httpResponse
               = Unirest.put("http://localhost:28139/upload")
            .header("Content-Type", "application/x-www-form-urlencoded")
            .asString();
    }
}
